doc/sphinx/source/utils.rst

```diff
@@ -288,3 +288,21 @@ Caveats
 
 - the tool doesn't yet work with derived variables; it will not return any available datasets;
```

Comment:
An additional suggestion for the doc/sphinx/source/community/diagnostic.rst section: may be add that one gets full provenance record only if the "full" esmvaltool run including preprocessor was done, and not only adding provenance part to the diagnostic on the pre-processed data? Feel free to ignore, if you think it's irrelevant.